src/dataset-builder/CohortEditor.ts

```diff
@@ -54,23 +55,25 @@ const addCriteriaText = 'Add criteria';
 export const CriteriaView = (props: CriteriaViewProps) => {
   const { datasetId, criteria, deleteCriteria, updateCriteria } = props;
 
-  const [criteriaCount, setCriteriaCount] = useLoadedData<DatasetParticipantCountResponse>();
+  const [criteriaCount, setCriteriaCount] = useLoadedData<SnapshotBuilderCountResponse>();
 
   const updateCriteriaCount = useRef(
     _.debounce(250, (datasetId, criteria) => {
       setCriteriaCount(
         async () =>
           await DataRepo()
             .dataset(datasetId)
-            .getCounts({
-              cohorts: [
-                {
-                  // Create a "cohort" to get the count of participants for this criteria on its own.
-                  criteriaGroups: [{ criteria: [criteria], name: '', meetAll: true, mustMeet: true }],
-                  name: '',
-                },
-              ],
-            })
+            .getSnapshotBuilderCount(
+              convertDatasetParticipantCountRequest({
+                cohorts: [
+                  {
+                    // Create a "cohort" to get the count of participants for this criteria on its own.
+                    criteriaGroups: [{ criteria: [criteria], name: '', meetAll: true, mustMeet: true }],
+                    name: '',
```

Comment:
I know this isn't your change, but I'm curious why is no name for the group or cohort saved?

Reply:
This API call is used to generate the `count` value for the criteria display. The `getCounts` API ignores names completely (it only looks at criteria) so they're not needed, and in this case there is no cohort name or cohort group name since we're making a fake one here just so we can get a rollup count separate from the overall cohort count.